source/parser.py

```diff
@@ -85,9 +85,9 @@ def authors(self):
                     aff = ', '.join([aff.strip() for aff in affiliation_text.split('\n') if aff.strip() != ''])
                     aff_list += [aff.strip()]
                 affs += [aff_list]
-        for i in range(len(authors_list)):
+        for i in range(min(len(authors_list), len(affs))):
```

Comment:
this can sovle the different length issue temporarily
The real issue is that authors list and affs list catch incorrect data, also there is data missing, working on it now.